crates/rust-fnv/sgx-tests/enclave/Makefile

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2017-2018 Baidu, Inc. All Rights Reserved.
```

Comment:
Copyright header is not needed. See the template. We want to make things consistent.